modules/stochastic_tools/test/tests/vectorpostprocessors/stochastic_results/tests

```diff
@@ -22,7 +22,28 @@
       cli_args = 'VectorPostprocessors/storage/parallel_type=DISTRIBUTED Outputs/file_base=distributed_out'
       csvdiff = 'distributed_out_storage_0002.csv.0 distributed_out_storage_0002.csv.1 distributed_out_storage_0002.csv.2'
 
-      detail = "distributed across many."
+      detail = "distributed across many,"
```

Comment:
Looks like this might have been left over, should revert back to previous language.